packages/taquito-utils/src/verify-signature.ts

```diff
@@ -61,7 +61,7 @@ function validateMessageNotEmpty(message: string) {
     return message;
 }
 
-function validatePkAndExtractPrefix(publicKey: string): PkPrefix {
+export function validatePkAndExtractPrefix(publicKey: string): PkPrefix {
     if (publicKey === '') {
         throw new InvalidPublicKeyError(`The public key provided for verifying signature cannot be empty.`);
```

Comment:
I suggest changing the error message to "The public key cannot be empty." as the precedent message doesn't make sense in the current context.

Reply:
will address!